lib/msal-node/docs/managed-identity.md

```diff
@@ -11,6 +11,7 @@ A common challenge for developers is the management of secrets, credentials, cer
 -   [Azure Arc](https://learn.microsoft.com/en-us/azure/azure-arc/overview)
 -   [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/overview)
 -   [Azure Service Fabric](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-overview)
+-   [Azure Machine Learning](https://azure.microsoft.com/en-us/products/machine-learning)
```

Comment:
do you need the locale here?

Reply:
yes

Reply:
you should not hardcode locale, 

see here for Azure India site :  https://azure.microsoft.com/en-in/products/machine-learning

Azure US : https://azure.microsoft.com/en-us/products/machine-learning

Azure Canada : https://azure.microsoft.com/en-ca/products/machine-learning

Australia : https://azure.microsoft.com/en-au/products/machine-learning 

based on the user region locale is auto detected

Reply:
please remove it from this PR and file a tracking item to remove locale from the codebase

Reply:
Sure: https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/7515